src/scenes/Login/Welcome.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
+import 'bootstrap/dist/css/bootstrap.min.css'
+import Login from './Login'
+import './welcome.scss'
+
+function Welcome () {
+  return (
+    <div className='welcome_login'>
+      <Router>
+        <Switch>
+          <Route exact path='/' component={Login} />
+        </Switch>
+      </Router>
+    </div>
+  )
+}
```

Comment:
why we need the Welcome component to wrap the Login component? 
Your anwser:

> i have one idea that: when i Login success , I will write code in Login component, includes: render Home.js. i think so!

Should note use this way to redirect to the home page.

Reply:
I removed Welcome.js, i forgot this component :>